l10n/en/firefox/new/desktop.ftl

```diff
@@ -8,8 +8,8 @@
 # the em dash is used in English to show a change of thought and
 # the em dash is used in meta titles to separate the title of a page from the title of the whole site
 # this title could also be written as "Download Firefox (Fast, Private & Free) — Mozilla"
-firefox-desktop-download-meta-title = Download { -brand-name-firefox-browser } — Fast, Private & Free — from { -brand-name-mozilla }
-firefox-desktop-download-meta-desc = Get { -brand-name-firefox }, a free web browser backed by { -brand-name-mozilla }, a non-profit dedicated to internet health and privacy. Available now on { -brand-name-windows }, { -brand-name-mac-short }, { -brand-name-linux }, { -brand-name-android } and { -brand-name-ios }.
+firefox-desktop-download-meta-title = Download { -brand-name-firefox } for Desktop — from { -brand-name-mozilla }
+firefox-desktop-download-meta-desc = Get { -brand-name-firefox } for { -brand-name-windows }, { -brand-name-mac-short } or { -brand-name-linux }. { -brand-name-firefox } is a free web browser backed by { -brand-name-mozilla }, a non-profit dedicated to internet health and privacy.
```

Comment:
We should create new strings here, rather than change the existing copy (so that translators will get notified that the source copy has changed).

Please create the new strings with the following IDs:

- `firefox-desktop-download-meta-title-v2`
- `firefox-desktop-download-meta-desc-v2`

The templates where the strings are used can then leverage the new strings, whilst falling back to the old ones where translations are not yet caught up e.g:

`{{ ftl('firefox-desktop-download-meta-title -v2', fallback='firefox-desktop-download-meta-title') }}`

Reply:
Hi @alexgibson , done . I have also added change to the hardcoded string for en-CA and en-GB in base.html. Please let me know if those are needed to be reverted.